src/connector/src/sink/kafka.rs

```diff
@@ -787,12 +708,14 @@ mod test {
         // Create the actual sink writer to Kafka
         let mut sink = KafkaSinkWriter::new(
             kafka_config.clone(),
-            schema,
-            pk_indices,
-            true,
-            "test_sink_1".to_string(),
-            "test_db".into(),
-            "test_table".into(),
+            SinkFormatterImpl::AppendOnlyJson(AppendOnlyFormatter::new(
+                JsonEncoder::new(
+                    schema.clone(),
+                    Some(pk_indices.clone()),
+                    TimestampHandlingMode::Milli,
+                ),
+                JsonEncoder::new(schema, Some(pk_indices), TimestampHandlingMode::Milli),
```

Comment:
The value/payload part should include all columns rather than only pk columns.

Reply:
Good catch! fixed.